etc/apache2/includes/instance-oo.include.template

```diff
@@ -77,8 +77,9 @@
     ProxyPassReverse        /elasticsearch       http://localhost:${DEPLOYMENT_ES_HTTP_PORT}
 
     # Forward request to OnlyOffice when the url start with the version scheme (see in side the Docker container for it)
-    # ex: /v5.2.8-24/*
+    # ex: /v5.2.8-24/* OR /5.3.0-243/* 
     ProxyPassMatch    "^\/(v[\d]+\.[\d]+\.[\d]+[\.|-][\d]+\/.*)$"       http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1 acquire=1000 retry=30
+    ProxyPassMatch    "^\/([\d]+\.[\d]+\.[\d]+[\.|-][\d]+\/.*)$"        http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1 acquire=1000 retry=30
```

Comment:
is there a reason to not improve the actual regexp to like ```"... ^\/(v?[\d]+...``` ?

Reply:
Yes, ignorance